advanced/events.md

```diff
@@ -171,12 +171,12 @@ tinymce.init({
 
 ### PreProcess
 
-This event gets fired when the contents inside the editor is being serialized to a HTML string.
+This event gets fired when the contents inside the editor are being serialized to an HTML string.
```

Comment:
revert to previous

Reply:
We will be using the "aytch" pronunciation.